sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileDataSourceV2.scala

```diff
@@ -49,6 +51,13 @@ trait FileDataSourceV2 extends TableProvider with DataSourceRegister {
   }
 
   protected def getTableName(paths: Seq[String]): String = {
-    shortName() + ":" + paths.mkString(";")
+    val name = shortName().capitalize + " " + paths.map(qualifiedPathName).mkString(",")
```

Comment:
`shortName().capitalize` looks like beyond the initial goal of this PR. Is this capitalization is required to pass the tests? Otherwise, shall we keep the existing behavior?